mldsa/src/sign.h

```diff
@@ -45,6 +45,8 @@
 /*************************************************
  * Hash algorithm enumeration for pre-hash functions
  **************************************************/
+#if !defined(MLD_CONFIG_MULTILEVEL_NO_SHARED)
+/* TODO: is there a better way?*/
```

Comment:
Put it in any KL-independent header (or a new one, `hash.h`)? This isn't great, but this is an internal header anyway, so it matters little. It's more of a nuisance in `mldsa_native.h`.